pkg/controller/mizar/mizar-service-controller.go

```diff
@@ -213,115 +214,143 @@ func (c *MizarServiceController) syncService(eventKeyWithType KeyWithEventType)
 
 func (c *MizarServiceController) processServiceCreation(service *v1.Service, eventKeyWithType KeyWithEventType) error {
 	key := eventKeyWithType.Key
-	netName := getArktosNetworkName(service.Name)
-	networkName, hasDNSServiceLabel := service.Labels[arktosapisv1.NetworkLabel]
+	tenant, namespace, name, err := cache.SplitMetaTenantNamespaceKey(key)
+	if err != nil {
+		return err
+	}
+
+	// Get tenant default network
+	tenantDefaultNetwork, err := c.networkLister.NetworksWithMultiTenancy(tenant).Get(defaultNetworkName)
+	if tenantDefaultNetwork.Spec.Type != mizarNetworkType {
+		return nil
```

Comment:
should this be an error case?

Reply:
> should this be an error case?

The design of tenant network object allows arktos to support multiple network solution. It is not an error case.